integrationtest/src/main/java/software/amazon/timestream/integrationtest/DatabaseMetaDataIntegrationTest.java

```diff
@@ -67,17 +67,53 @@ private void terminate() throws SQLException {
     connection.close();
   }
 
+  /**
+   * Test getCatalogs returns empty ResultSet.
+   * @throws SQLException the exception thrown
+   */
   @Test
-  @DisplayName("Test retrieving all databases.")
+  @DisplayName("Test getCatalogs(). Empty result set should be returned")
   void testCatalogs() throws SQLException {
-    final List<String> databasesList = Arrays.asList(Constants.DATABASES_NAMES);
     final List<String> catalogsList = new ArrayList<>();
     try (ResultSet catalogs = metaData.getCatalogs()) {
       while (catalogs.next()) {
         catalogsList.add(catalogs.getString("TABLE_CAT"));
       }
     }
-    Assertions.assertTrue(catalogsList.containsAll(databasesList));
+    Assertions.assertTrue(catalogsList.isEmpty());
+  }
+
+  /**
+   * Test getSchemas returns the list of all databases.
+   * @throws SQLException the exception thrown
+   */
+  @Test
+  @DisplayName("Test retrieving all databases.")
+  void testSchemas() throws SQLException {
+    final List<String> databasesList = Arrays.asList(Constants.DATABASES_NAMES);
+    final List<String> schemasList = new ArrayList<>();
+    try (ResultSet schemas = metaData.getSchemas()) {
+      while (schemas.next()) {
+        schemasList.add(schemas.getString("TABLE_SCHEM"));
+      }
+    }
+    Assertions.assertTrue(schemasList.containsAll(databasesList));
+  }
+
+  /**
+   * Test getSchemas returns database "JDBC_Integration07_Test_DB" when given matching patterns.
+   * @param schemaPattern the schema pattern to be tested
+   * @throws SQLException the exception thrown
+   */
+  @ParameterizedTest
+  @ValueSource(strings = {"JDBC_%", "%_Integration%", "%Test_DB"})
```

Comment:
In SQL standard, the `_` is also used as single character wildcard pattern, we should add specific test cases for it as well. Currently `_` just happens to be included in the database name itself.

Reply:
SQL standard also allows customers to specify the escape character in patterns with `ESCAPE` clause. Can we evaluate if it fits passing down to schema-pattern in a JDBC driver, in which case, some tests will need to be added.

Example:
```
SHOW DATABASES LIKE 'A3!_showDatabaseTestWithNoWildcard_' ESCAPE '!'"
```

Reply:
I have added testcase for "_" and escape character using escape clause.